tests/integration/tests/test_util/snap.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright 2024 Canonical, Ltd.
+#
+import json
+import logging
+import re
+import urllib.error
+import urllib.request
+from typing import List
+
+LOG = logging.getLogger(__name__)
+
+SNAP_NAME = "k8s"
+
+# For Snap Store API request
+SNAPSTORE_API = "https://api.snapcraft.io/v2/snaps/info/"
```

Comment:
maybe we can change this to `SNAPSTORE_INFO_API`? cause I feel like snapstore api should be a more general endpoint.